__tests__/integration/nestedResources.test.js

```diff
@@ -8,36 +8,64 @@ describe('Expanding a resource property in a property panel', () => {
+    expect.assertions(3)
+    await pupExpect(page).toClick('a[data-id=\'title\']')
+    await pupExpect(page).toClick('a[data-id=\'mainTitle\']')
+    await pupExpect(page).toMatchElement('input[placeholder=\'Preferred Title for Work (RDA 6.2.2, RDA 6.14.2) (BIBFRAME: Main title)\']')
   })
 
-  it('enters a value into a nested prooperty component', async () => {
+  it('enters a title into the BIBFRAME Instance Work Title component', async () => {
     expect.assertions(3)
-    await pupExpect(page).not.toMatchElement('div#userInput', { text: 'Some text' })
-    await expect(page).toFill('input[placeholder="Holdings"', 'Some text')
+    await pupExpect(page).not.toMatchElement('div#userInput', { text: 'An interesting title' })
+    await expect(page).toFill('input[placeholder="Preferred Title for Work (RDA 6.2.2, RDA 6.14.2) (BIBFRAME: Main title)"', 'An interesting title')
     await page.keyboard.press('Enter')
-    await pupExpect(page).toMatchElement('div#userInput', { text: 'Some text' })
+    await pupExpect(page).toMatchElement('div#userInput', { text: 'An interesting title' })
   })
 
-  it('enters a non-roman value into a nested property component', async () => {
+  it('clicks on the Responsibility Statement property to reveal the responsibility input component', async () => {
     expect.assertions(2)
-    await pupExpect(page).toFill('input[placeholder="Holdings"', '甲骨文')
+    await pupExpect(page).toClick('a[data-id=\'responsibilityStatement\']')
+    await pupExpect(page).toMatchElement('input[placeholder=\'Statement of Responsibility Relating to Title Proper (RDA 2.4.2)\']')
+  })
+
+  it('enters text into the BIBFRAME Instance Statement of Responsibility component', async () => {
+    expect.assertions(3)
+    await pupExpect(page).not.toMatchElement('div#userInput', { text: 'A statement of responsibility' })
+    await expect(page).toFill('input[placeholder="Statement of Responsibility Relating to Title Proper (RDA 2.4.2)"', 'A statement of responsibility')
     await page.keyboard.press('Enter')
-    await pupExpect(page).toMatchElement('div#userInput', { text: '甲骨文' })
+    await pupExpect(page).toMatchElement('div#userInput', { text: 'A statement of responsibility' })
+  })
+
+  it('clicks on Contribution property to reveal the input component', async () => {
+    expect.assertions(6)
+    await pupExpect(page).toClick('a[data-id=\'contribution\']')
+    await pupExpect(page).toMatchElement('input[placeholder=\'Agent Contribution\']')
+    await expect(page).toFill('input[placeholder=\'Agent Contribution\']', 'Stanford family')
+    await pupExpect(page).toMatchElement('ul#lookupComponent')
+    await page.waitForSelector('li#rbt-menu-item-0')
+    await pupExpect(page).toMatchElement('li#rbt-menu-item-0')
+    await pupExpect(page).toClick('li#rbt-menu-item-0')
+  })
+
+  it('it clicks on Preview RDF and verifies the text', async () => {
```

Comment:
i really like adding this, showing that entered text at least makes it to the preview.  I'm pondering something organized more like this (but again, it's certainly no worse that what was there before you added stuff to it):

```
beforeAll() - bunch of setup
it can expand a nested resource (property type "resource") ... is it 1 level deep or 2?
it can add a nested literal property (x levels deep) and it shows up in preview RDF
it can add a nested lookup property (x levels deep) and it shows up in preview RDF
```